src/cmd/build.js

```diff
@@ -37,14 +39,48 @@ export type PackageCreatorParams = {
   artifactsDir: string,
 };
 
+export type LocalizedNameParams = {
+  messageFile: string,
+  manifestData: ExtensionManifest,
+}
+
+export async function getDefaultLocalizedName(
+  {messageFile, manifestData}: LocalizedNameParams
+): Promise<string> {
+
+  let messageData: string|Buffer;
+  let extensionName: string = manifestData.name;
+
+  try {
+    messageData = JSON.parse(await fs.readFile(messageFile));
+  }
+  catch (error) {
+    throw new UsageError(
+      `Error reading or parsing file ${messageFile}: ${error}`);
+  }
+  extensionName = manifestData.name.replace(/__MSG_([A-Za-z0-9@_]+?)__/g,
+    (match, messageName) => {
+      if (!(messageData[messageName]
+            && messageData[messageName].message)) {
+        const error = new UsageError(
+          `The locale file ${messageFile} ` +
+            `is missing key: ${messageName}`);
```

Comment:
I see that you indented this to show continuation but it actually wouldn't need indenting since it's not a *new* block. It's just an extension of the argument being passed to `UsageError()`.

No need to fix this since it's so minor, just something to keep in mind.